packages/fast-stable-stringify/src/__tests__/json_test.ts

```diff
@@ -3,7 +3,7 @@ import stringify from '..';
 describe('test_fast_stable_stringify', function () {
     it('big int', function () {
         const obj = { age: 23, bigint: BigInt('200'), name: 'ABCD' };
-        expect(stringify(obj)).toMatch('{"age":23,"bigint":"200n","name":"ABCD"}');
+        expect(stringify(obj)).toMatch('{"age":23,"bigint":"200n‽¶","name":"ABCD"}');
```

Comment:
If you put the glyph _outside_ the quotes, you can use a simpler character, and you will never get a collision.

Consider:

```ts
s(100n) // "100"n
s({foo: 100n}) // {"foo":"100"n}
```